### PR TITLE
EOL'd ServiceOverlay. Added Open Service Button to open in new tab

### PR DESCRIPTION
**Pull request by MatApple**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @jfurrow

File: src/js/components/ServiceOverlay.js [hunk 1]
```diff
@@ -1,218 +0,0 @@
-import mixin from 'reactjs-mixin';
```
Comment (kennyt) [inline src/js/components/ServiceOverlay.js]: this was the first component i ever created for mesosphere. RIP

Comment (MatApple) [inline src/js/components/ServiceOverlay.js]: Sad day for all..

**Comment by jfurrow** [comment]
🚀

**Comment by daiyiwhale** [comment]
⛵

File: src/js/components/ServiceOverlay.js (hunk 1 above)
Comment (daiyiwhale) [inline src/js/components/ServiceOverlay.js]: 💀

**Pull request merged**